### ISSUE TITLE
Tomb support (it's just LUKS with a little bit more)

### ISSUE THREAD

**Issue by jaromil** [t-d-k/doxbox#9]
Hi there, I'm Tomb's developer, see https://github.com/dyne/tomb

Our tool is widely used on gnu/linux platform and I'm sure many people would find it handy to open tombs also on win. Would you be interested in adding support to open tombs in doxbox? It should be simple, since you have already LUKS covered: Tombs are LUKS volumes whose keys are password-protected files symmetrically encrypted with GnuPG (and optionally steg-hidden inside JPG images).

In any case, thanks for your effort!

**Comment by t-d-k** [comment]
Thanks for this suggestion, I think there is more support for adding other formats like Truecrypt before Tomb. DoxBox already supports LUKs key files, and this wouldn't add much over that apart from compatibility.

**Comment by jaromil** [comment]
Yep, I understand. It's just a low-hanging fruit in terms of compatibility with another user-base.
Perhaps I'll manage to contribute it myself later. Cheers

**Comment by linux-modder** [comment]
@jaromil ,   I'll fork  your branch and see if I can't get it to add-on if only a bootstrap at first.   are you wanting basic tomb or nested (requiring shim on efi)

**Comment by jaromil** [comment]
hi! by nested what do you mean? Slam does `fuser` and ` kill -9` of all processes using the mounted fs. Not sure how we can get that across platforms, `vgchange` I've never heard of, but once you have a draft of the changes needed I'll do my best to help. The idea is that this task should be trivial, but lets see.

**Comment by linux-modder** [comment]
Nested :  
sda                                          8:0    0 232.9G  0 disk  
├─sda1                                       8:1    0     2M  0 part  
├─sda2                                       8:2    0   512M  0 part  /boot
└─sda3                                       8:3    0 232.4G  0 part  
  └─luks-10740d32-007a-4492-b51a-deb3ba470886
                                           253:0    0 232.4G  0 crypt 
    ├─secure_fedora-root                   253:1    0    10G  0 lvm   /
    ├─secure_fedora-usr                    253:2    0    15G  0 lvm   /usr  
    ├─secure_fedora-swap                   253:3    0     5G  0 lvm   [SWAP]
                  ├─secure_fedora-var_www                253:8    0     5G  0 lvm   /var/www     (nested)
                  ├─secure_fedora-opt                    253:9    0     5G  0 lvm   /opt       (nested)
                  ├─secure_fedora-srv                    253:10   0     5G  0 lvm   /srv      (nested)
                  └─secure_fedora-home       (nested)

**Comment by jaromil** [comment]
just a heads up if there is anything I can do to help with this, from testing to code, let me know. I really believe basic support for open/close and perhaps even slam is trivial to implement. As of steganography I can try to compile steghide for Win platform if you like. Regarding issue #22 we are fine, Tomb does not use GBT nor LVM for its loopback partitions.
